### DISCUSSION TITLE
How to Make Contributions to Pulsar Documentation and Website?

### DISCUSSION THREAD

**Discussion by Huanli-Meng** [apache/pulsar#17329] [Q&A] [answered]
Hi team,
I want to update Pulsar documentation and website, are there any guides to reference or standards to follow?
Thank you!

**Comment by Anonymitaet** [comment, marked as answer]
Thanks for raising this up!

The [Pulsar Contribution Guide](https://pulsar.apache.org/contributing/) lists all the following guides explaining how to make contributions to Pulsar documentation and website.

☘️☘️☘️

🔎 **Before writing documentation** 

- [Pulsar Documentation Contribution Guide](https://docs.google.com/document/d/1-1uJyd1k9_h56xiiVRVOnrLcCnTmg9n7SrHhNVNEEi4/edit#heading=h.f4ftjqe2pezi)

☘️☘️☘️

✏️ **Writing documentation** 

- [Pulsar Documentation Writing Style Guide](https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#)
- [Pulsar Documentation Writing Syntax Guide](https://docs.google.com/document/d/12De2btkDHQVaqUlHjTqERmroMLKGhHdiC7rEttFTyqc/edit#) 
- [Pulsar Design Style Guide](https://docs.google.com/document/d/16Hp7Sc86MQtL0m8fc2w_TrcKXAuglwRwHmdmwfk00mI/edit#heading=h.b8ogodj5sj0)
- [Pulsar API Documentation Guide](https://docs.google.com/document/d/1-I1oQp1_HUaQopqilU-JdC-ksrLAgYNi93FZVnECwV8/edit#heading=h.wu6ygjne8e35)
- [Pulsar Release Notes Guide](https://docs.google.com/document/d/1cwNkBefKyV6OPbEXnUrcCdVZi0i2BezqL6vAL7VqVC0/edit#)

☘️☘️☘️ 

🧪 **Testing documentation**

- [Pulsar Documentation Preview Guide](https://docs.google.com/document/d/1wszdtMRo6MhKbVaggPK7_bnKaC4TewuT--GWZZxJNGg/edit#heading=h.tng8x5vsxlpi)

☘️☘️☘️ 

🚀 **Preparing to submit a documentation PR** 

- [Pulsar Pull Request Naming Convention Guide](https://docs.google.com/document/d/1d8Pw6ZbWk-_pCKdOmdvx9rnhPiyuxwq60_TrD68d7BA/edit#heading=h.wu6ygjne8e35)
- [Pulsar Documentation Label Guide](https://docs.google.com/document/d/1Qw7LHQdXWBW9t2-r-A7QdFDBwmZh6ytB4guwMoXHqc0/edit#)

☘️☘️☘️

💡 **Tip** 
- These guides are in drafts and continuously updated. More specific guides are on the way. Feel free to comment!
- Do not worry about the accessibility. They will be **moved to the Pulsar website** after they are finalized.
